spec/api/v2/admin/users_spec.rb

```diff
@@ -55,6 +56,26 @@ def validate_fields(user)
         expect(response.headers.fetch('Per-Page')).to eq '100'
       end
 
+      let(:extended_params) do {
+          extended: true
+        }
+      end
+      let(:do_extended_info_request) { get '/api/v2/admin/users', headers: auth_header, params: extended_params }
+
+      it 'returns list of users with full info' do
+        do_extended_info_request
+        users = JSON.parse(response.body)
+        expect(User.count).to eq users.count
+
+        expect(users.first['labels']).to eq []
+        expect(users.first['activities']).to eq []
```

Comment:
We need remove activities from this call.

Reply:
@calj  So we should remove them in entity?

Reply:
@gfedorenko no, we need to add additional entity.
Activities inside this entity used in several endpoints, so we cant remove them